launcher/ui/pages/modplatform/modrinth/ModrinthPage.cpp

```diff
@@ -60,9 +61,19 @@ ModrinthPage::ModrinthPage(ModDownloadDialog* dialog, BaseInstance* instance)
     connect(ui->modSelectionButton, &QPushButton::clicked, this, &ModrinthPage::onModSelected);
 }
 
-auto ModrinthPage::validateVersion(ModPlatform::IndexedVersion& ver, QString mineVer, QString loaderVer) const -> bool
+auto ModrinthPage::validateVersion(ModPlatform::IndexedVersion& ver, QString mineVer, ModAPI::ModLoaderType loader) const -> bool
 {
-    return ver.mcVersion.contains(mineVer) && ver.loaders.contains(loaderVer);
+    auto loaderStrings = ModrinthAPI::getModLoaderStrings(loader);
+
+    auto loaderCompatible = false;
+    for (auto loader : ver.loaders)
```

Comment:
Can you change this `loader` to something else? It shadows (is this the right term?) the function parameter, and makes it harder to see which one is being referenced later on imo